### DISCUSSION TITLE
What is the event stream switch for?

### DISCUSSION THREAD

**Discussion by ajma** [tsightler/ring-mqtt#572] [Support Q&A] [answered]
I have some ring cameras using this add-on but I'm unclear on what the event stream switch for? It's off but what does it do if I turn it on?

**Comment by tsightler** [comment, marked as answer]
It's mostly just to indicate if there is a current event stream viewer or not, i.e. any RTSP client connected to the event stream.  It can be used to force stop a stream, but if the client reconnects then it will simply turn back on, altought the event playback will restart from the beginning.

If you attempt to turn on the event stream manually, without any clients, it will just log a message that this isn't possible and turn back off.
